controllers/companies/index/get.js

```diff
@@ -0,0 +1,27 @@
+const Company = require('../../../models/company/Company');
+
+module.exports = (req, res) =>{
+
+  //default return all companies
+  Company.findCompaniesByFilter(req.query, req.query, (err, data) =>{
+    if(err || data == 'database_error') {
+      console.log(err)
+      res.redirect('/companies');
+    }
+
+  return res.render('companies/index', {
```

Comment:
data database_error a eşit olamaz, fonksiyonda öyle bir şey varsa onu error olarak vericek şekilde düzelt. Bir error'un error field hariç bir yerde bulunmaması gerekiyor. return bir tab içerde olacak ve sonlarına noktalı vürgül gelecek ama çok önemli değil.

Reply:
Aslında kodu, buna göre yazmıştım ama burada yanlışlıkla yazmıştım data == "database_error", ama senin yaptığın callback(error, returnObject) olayını anladım, daha doğrusu callback(null, returnObject) ya da callback(error)